acms-react-app/src/Services/auth.service.js

```diff
@@ -1,50 +1,30 @@
 import axios from 'axios';
 
-const API_URL = 'http://localhost:8080/api/v1/auth/';
+const API_URL = `${process.env.REACT_APP_BACKEND_URL || 'http://localhost:8080'}/api/v1/auth/`;
 
 class AuthService {
-
     googleLogin(token){
-        return axios.post(API_URL + 'google-login/' + token)
-            .then(response => {
-                if(response.data.token){
-                    sessionStorage.setItem('user',JSON.stringify(response.data));
-                }
-                return response.data;
-            });
+        return axios.post(`${API_URL}google-login/${token}`)
+            .then(this.handleResponse);
```

Comment:
Good job on noticing the repeatable code and creating this handle response method that keeps everything cleaner